cli/pkg/local/install_id.go

```diff
@@ -63,11 +63,11 @@ func config() (*localConfig, error) {
 	if conf.InstallID == "" {
 		// create install id if not exists
 		conf.InstallID = uuid.New().String()
-		confJson, err := json.Marshal(&conf)
+		confJSON, err := json.Marshal(&conf)
 		if err != nil {
 			return conf, err
 		}
-		err = os.WriteFile(confFile, confJson, 0644)
+		err = os.WriteFile(confFile, confJSON, 0o644)
```

Comment:
What rule is this? The 0644 isn't great, but this change just makes it weirder

Reply:
Yeah thats right, this is weird, this rule is removed but i guess forgot to revert this file